app/templates/events/view/tickets/discount-codes.hbs

```diff
@@ -0,0 +1,43 @@
+<div class="ui hidden divider"></div>
+<div class="content">
+  <button class="ui teal button right floated">
+    <i class="plus icon"></i>
+    {{t 'Create a discount'}}
+  </button>
+</div>
+<div class="content">
+  <h2 class="ui header">{{t 'Discount Codes'}}</h2>
+</div>
+<div class="ui hidden divider"></div>
+<div class= "content">
+  <div class="ui stackable basic buttons">
+    <div class="ui button active">{{t 'All'}}</div>
+    <div class="ui button">{{t 'Active'}}</div>
+    <div class="ui button">{{t 'Inactive'}}</div>
+  </div>
+  <table class="ui celled table">
+    <thead>
+      <tr>
+        <th>#</th>
+        <th>{{t 'Discount code'}}</th>
+        <th>{{t 'Discount code URL'}}</th>
+        <th>{{t 'Discount Per Ticket'}}</th>
+        <th>{{t 'Validity'}}</th>
+        <th>{{t 'Status'}}</th>
+        <th>{{t 'Action'}}</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td></td>
+        <td></td>
+        <td></td>
+        <td>{{t 'No data available in Table'}}</td>
```

Comment:
Please fill this with actual data, use orga server for reference to see how the table looks with data filled in.